src/status_im/chat/views/input/input.cljs

```diff
@@ -159,7 +159,8 @@
                               :editable            (not @sending-in-progress?)
                               :on-focus            #(dispatch [:set-chat-ui-props {:show-emoji? false}])
                               :on-submit-editing   (fn []
-                                                     (when-not (or (str/blank? @seq-arg-input-text)
+                                                     (when-not (or (and (str/blank? @seq-arg-input-text)
+                                                                        (not= "password" (get-in @command [:command :name])))
```

Comment:
Relying on `command name` sounds fragile to me.
Maybe it's worth looking at some separate attribute to encapsulate this behavior.